### ISSUE TITLE
t/reactor_epoll.t fails

### ISSUE THREAD

**Issue by eserte** [Grinnz/Mojo-Reactor-Epoll#1]
Another epoll problem, maybe also related to Net::DNS::Native:
```
Couldn't add filehandle from epoll set: Bad file descriptor at /tmpfs/.cpan-build-cpansand/2018022415/Mojo-Reactor-Epoll-0.006-6LayOw/blib/lib/Mojo/Reactor/Epoll.pm line 152.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 9 just after 77.
destroying Net::DNS::Native object while queue for resolver has 1 elements during global destruction.
destroying Net::DNS::Native object with 1 non-received results during global destruction.
t/reactor_epoll.t ...... 
Dubious, test returned 9 (wstat 2304, 0x900)
All 77 subtests passed 
```

**Comment by Grinnz** [comment]
This appears to occur if Net::DNS::Native 0.16-0.18 is installed, it's fine on 0.15. As a temporary workaround I'm disabling the use of Net::DNS::Native in the tests.

**Comment by Grinnz** [comment]
I've opened https://rt.cpan.org/Ticket/Display.html?id=124932 for Net::DNS::Native

**Comment by Grinnz** [comment]
According to oleg, this is an issue with Linux::Epoll not being able to handle the tied handle Net::DNS::Native now uses. I've opened https://rt.cpan.org/Ticket/Display.html?id=124934 for Linux::Epoll.

**Comment by Grinnz** [comment]
This should be fixed by Linux::Epoll 0.013, which will be required by the next release. Thanks!
